apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -18,11 +18,13 @@ import { AppLogger } from '../logger/logger.service';
 
 @Injectable()
 export class CashDepositService {
+  private readonly appLogger;
   constructor(
-    @Inject(Logger) private readonly appLogger: AppLogger,
     @InjectRepository(CashDepositEntity)
     private cashDepositRepo: Repository<CashDepositEntity>
-  ) {}
+  ) {
+    this.appLogger = new AppLogger(CashDepositService.name);
```

Comment:
In this PR, we are instantiating appLogger for every class. Which I think is an okay solution, but I do prefer the injectable I think, https://docs.nestjs.com/techniques/logger#injecting-a-custom-logger